assets/css/errors.scss

```diff
@@ -0,0 +1,20 @@
+.not_found {
+  position: absolute;
+  top: 50%;
+  margin-left: 50%;
+  padding: 0.5rem 0.75rem;
+  transform: translateY(-50%) translateX(-50%);
+  text-align: center;
+  color: $purple;
+
+  &__code {
```

Comment:
throughout the project we don't use the `&__` notation, so to keep it consistent with every thing else, can you change it to `&-` please?